### ISSUE TITLE
Spark源码阅读(九)：存储体系之内存存储

### ISSUE THREAD

**Issue by wzx140** [wzx140/blog-comment#41]
https://masterwangzx.com/2020/09/09/store-memory/ 

 介绍Spark中的block内存存储。Hadoop只将内存作为计算资源，Spark除将内存作为计算资源外，还将内存的一部分纳入到存储体系中。

**Comment by iam91** [comment]
大佬，我理解MemoryManager不止是用于管理存储部分的内存吧？应该是存储内存和执行内存都由它管理？

**Comment by wzx140** [comment]
> @iam91
> 大佬，我理解MemoryManager不止是用于管理存储部分的内存吧？应该是存储内存和执行内存都由它管理？

是的。UnifiedMemoryManager维护执行内存和存储内存之间的软边界，执行内存可以看 [Spark源码阅读(十九)：计算引擎之task计算内存管理](https://masterwangzx.com/2020/09/24/calculation-memory/)
